projects/plugins/jetpack/modules/likes.php

```diff
@@ -425,19 +425,8 @@ function_exists( '\Activitypub\is_activitypub_request' )
 		* If the same post appears more then once on a page the page goes crazy
 		* we need a slightly more unique id / name for the widget wrapper.
 		*/
-		$uniqid = uniqid();
-		/**
-		 * Enable an alternate Likes layout.
-		 *
-		 * @since 12.9
-		 *
-		 * @module likes
-		 *
-		 * @param bool $new_layout Enable the new Likes layout. False by default.
-		 */
-		$new_layout = apply_filters( 'likes_new_layout', true ) ? '&amp;n=1' : '';
```

Comment:
Note that this filter is still used on WP.com and we might want to clean it up.

Good news is that it was always set to return `true`.